packages/commands/src/components/command-menu.js

```diff
@@ -158,25 +158,23 @@ export function CommandMenu() {
 			description: __( 'Open the command palette' ),
 			keyCombination: {
 				modifier: 'primary',
-				character: 'k',
+				character: '/',
 			},
+			aliases: [ { character: 'F1' } ],
 		} );
 	}, [ registerShortcut ] );
 
-	useShortcut(
-		'core/commands',
-		( event ) => {
-			event.preventDefault();
-			if ( isOpen ) {
-				close();
-			} else {
-				open();
-			}
-		},
-		{
-			bindGlobal: true,
+	useShortcut( 'core/commands', ( event ) => {
+		// Bails to avoid obscuring the effect of the preceding handler(s).
+		if ( event.isDefaultPrevented() ) return;
```

Comment:
I believe `isDefaultPrevented()` is [a jQuery event object](https://api.jquery.com/event.isdefaultprevented/). Since the document loads jQuery, this function would work, but I think `defaultPrevented` is better to match the other implementations.

Reply:
Oh, I guess it's not a coincidence React [has the same method](https://react.dev/reference/react-dom/components/common#react-event-object-methods) on its events. Here, `event` is a `SyntheticEvent` and this should be fine. To double-check, the Provider which attaches the handler that is the source of these events:
https://github.com/WordPress/gutenberg/blob/33bf984897683adfc4582a29adb079cf6ee0ab59/packages/keyboard-shortcuts/src/components/shortcut-provider.js#L33-L34

I think it’s easy to get `useShortcut` mixed up with `useKeyboardShortcut`. E.g. 33f6f21. If using the latter, one would indeed need to go with `defaultPrevented`.

Reply:
I see, I did not know React had the method 😅